src/runtime/hexagon_remote/halide_hexagon_remote.idl

```diff
@@ -4,11 +4,12 @@ interface halide_hexagon_remote
     typedef unsigned long handle_t;
 
     // Routine to initialize a module on the remote side. Returns a pointer to the module.
-    long initialize_kernels(in buffer code, rout handle_t module_ptr);
+    long initialize_kernels(in buffer code, in long use_dlopen, in long use_dlopenbuf,
```

Comment:
These functions can't just be changed. You need to add a new version of these functions (e.g. initialize_kernels_v2), and possibly build a shim in halide_host.cpp (note the wrapper for get_symbol around get_symbol_v2).

In this case, you should probably add an initialize_kernels_v2 and get_symbol_v3 that take usedl, and you can add shims for initialize_kernels and get_symbol_v2 that pass false for usedl to the new versions.